src/guide/1/dbsearch/javascript/snippets/final.js

```diff
@@ -0,0 +1,62 @@
+// Require and instanciate kuzzle
```

Comment:
```suggestion
// Require and instantiate kuzzle
```

(but I would remove that comment: it explains how JS works, not how Kuzzle works)